clusterbuster

```diff
@@ -2132,6 +2201,19 @@ function _log_container() {
     fi
 }
 
+function _retrieve_vm_xml() {
+    local namespace=$1
+    local vm=$2
+    local clog="$artifactdir/VMXML/${namespace}_${vm}.xml"
+    if __OC exec -n "$namespace" "$(__OC get pod -n "$namespace" -l "vm.kubevirt.io/name=$vm" '-ojsonpath={.items[0].metadata.name}')" -- /bin/sh -c "cat '/var/run/libvirt/qemu/run/${namespace}_${vm}.xml'" > "${clog}.tmp" ; then
```

Comment:
By the way, `oc rsh -h` is getting more popular these days.

Reply:
I'm considering that.  At present it's succeeding, but the issue of `oc exec` not working in privileged pods is something I should think about.